pyuvdata/uvbase.py

```diff
@@ -232,6 +240,9 @@ def check(self, check_extra=True, run_check_acceptability=True):
                 otherwise only check required parameters.
             run_check_acceptability: Option to check if values in required parameters
                 are acceptable. Default is True.
+            only_check_set: Option to check only that values are acceptable for parameters
+                that have been set. This is equivalent to run_check_acceptability, but will
```

Comment:
I don't think this docstring is quite right. `run_check_acceptability` checks whether the parameters have acceptable values, the basic check is what checks that the shapes are correct (and that the values aren't None). This code changes the basic check, not the acceptability check. Feel free to improve the docstring for `run_check_acceptability` if it's not clear enough.

Reply:
I've been having trouble determining how best to phrase this. It's mostly useful to the acceptability check, since it lets the user check that parameters are acceptable even when the UVData is incomplete. But it is a change to the basic check. In a way, it's separating the acceptability check from the requirements check.

I've renamed the keyword to `ignore_requirements` and tried to better explain the use for it in the docstring.

Reply:
But if you *don't* set `run_check_acceptability`, it still lets you check that the shapes are right, which you can't without this option.

Reply:
Oh, I think I follow. So the acceptability check just checks that the values make sense, but the basic check also looks at shapes?

Reply:
Yes. The basic check checks shapes and data types. The acceptibility check checks that the values are within specified ranges.

Reply:
Apparently the docstring isn't clear enough, can you update it to make it clearer?